packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusableChildrenGrid.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Flex,
+  Text,
+  Card,
+  Grid,
+  cardsContainerBehavior,
+  cardChildrenFocusableBehavior,
+  Button,
+  StarIcon,
+  DownloadIcon,
+  MoreIcon,
+} from '@fluentui/react-northstar';
+import { screenReaderContainerStyles } from '@fluentui/react-northstar';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+type SampleCardProps = {
+  index?: number;
+};
+
+const SampleCard: React.FC<SampleCardProps> = ({ index, ...unhandledProps }) => {
+  return (
+    <Card accessibility={cardChildrenFocusableBehavior} aria-roledescription="user card" {...unhandledProps}>
+      <Card.Header>
+        <Text content={`User #${index}`} weight="bold" />
+      </Card.Header>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+  );
+};
+
+const CardExampleFocusableGrid = () => {
+  const navigationMessageId = 'navigationMessage';
+  const defaultMessage = 'Press arrow keys to navigate between cards.';
+  const [navigationMessage, setNavigationMessage] = React.useState(defaultMessage);
+  const timeout = React.useRef<ReturnType<typeof setTimeout>>();
+  const cardsNumber = 18;
+  const handleGridFocus = () => {
+    if (navigationMessage !== null) {
+      timeout.current = setTimeout(() => {
+        setNavigationMessage(null);
+      }, 3000);
+    }
+  };
+
+  const handleGridBlur = e => {
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setNavigationMessage(defaultMessage);
+    }
+  };
+
+  const cleanup = () => {
+    if (timeout.current) {
+      clearTimeout(timeout.current);
+    }
+  };
+
+  React.useEffect(() => cleanup, []);
+
+  return (
+    <>
+      <div id={navigationMessageId} style={screenReaderContainerStyles}>
+        {navigationMessage}
+      </div>
+      <Grid accessibility={cardsContainerBehavior} columns="3" onFocus={handleGridFocus} onBlur={handleGridBlur}>
+        {_.times(cardsNumber, i => {
+          const nonZeroIndex = i + 1;
+          return (
+            <SampleCard
```

Comment:
I tried just now with NVDA
![image](https://user-images.githubusercontent.com/22620150/80210263-bff24080-8633-11ea-86ae-43ba14d75460.png)
and the card is missing label. I mean label in the way it narrate only "aria-roledescription" and "position". 

Please extend aria-label itself the easiest way probably. Please put it before the position in the string. 

Or you can use aria-labelledby="(userID) (cardID)" > in this structure it would narrate user name which is displayed in the card and then the aria-label + aria-roledescription + aria+describedby  from the card itself.
But you would need to add ID for user element and ID for card itself.

Reply:
Updated `aria-labelledby` with ids, tested in VoiceOver, NVDA and JAWS, thanks @kolaps33